docs/api-documentation/rpc.md

```diff
@@ -116,5 +155,36 @@ Note, this is Tendermint-like compatibility, that should be refactored.
 
 `tx(hash: bytes)`: queries status of the transaction by hash, returns FinalTransactionResult that includes status, logs and result: `{"status": "Completed", "logs": [{"hash": "<hash>", "lines": [], "receipts": [], "result": null}]}`
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx params:=["<base 58 of transaction hash>"] id=dontcare`
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx id=dontcare 'params:=["<base 58 of transaction hash>"]'`
 
+```javascript
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": {
+        "status": "Completed",
+        "transactions": [
+            {
+                "hash": "2ZBygTrowP7MTeRQ8rXp8v84benMTaYbHhwFw5w85CTS",
+                "result": {
+                    "logs": [],
+                    "receipts": [
+                        "CdbagwttuZnTcjGT3R8Qs9fsv8GFuFeUtJVnVZeAjoop"
+                    ],
+                    "result": null,
+                    "status": "Completed"
```

Comment:
Can it be in any state but "completed"?

Reply:
Given we can get a list of transactions only by querying block/chunk, it seems that we won't be able to catch a non-complete transaction... Well, it does not hurt having this field anyway.